workbench/icons/Makefile.am

```diff
@@ -0,0 +1,20 @@
+icondir = $(datadir)/icons/hicolor/16x16/apps
```

Comment:
Well, then I think I will go for the single makefile approach suggested by @b4n because it seems to be the simplest solution.

Another thing regarding icons:
I noticed that there is no 16x16 icon for javascript files. That means if I have a javascript file listed in a workbench project directory, then it is bigger than the other icons. I assumed there would be references to icon files for each extension in geany but I did not find it. Is this coming from the OS? So in my case Ubuntu/Gnome. Should I open an issue there about the missing 16x16 icon for javascript?